### ISSUE TITLE
Wad2bin Error

### ISSUE THREAD

**Issue by KagamineRin271** [DarkMatterCore/wad2bin#26]
I'm always getting the message "Paths must not exceed 259 characters" when trying to convert something. How do I fix this (I use a Windows 11 and a 128 SD card)?

**Comment by Kurty00** [comment]
The path to your files is too long. Try moving the entire thing up in your folder structure a bit.

**Comment by KagamineRin271** [comment]
> The path to your files is too long. Try moving the entire thing up in your folder structure a bit.

How?

**Comment by KagamineRin271** [comment]
What do I need to move?

**Comment by KagamineRin271** [comment]
Do I move my Wad folder on my SD card or move my Wad2bin folder?

**Comment by dippyshere** [comment]
> How?

The number of characters that make up the path (e.g. `C:/folder/path/file`) cannot exceed 259 for reasons.
> What do I need to move?

Move your source wads up in the folder structure (e.g. instead of `C:/folder/folders/blahblah/wads`, you can do `C:/wads/`).
> Do I move my Wad folder on my SD card or move my Wad2bin folder?

Since we don't know your setup / paths you are trying, you can try some of these general tips for file / folder structure:
- Set the SD card path to the root of your SD card; The WADs will be put in the root of your SD card as `TITLEID_bogus.wad`
- Try putting your WADs, wad2bin and the wad2bin GUI script all inside of a folder at the root of a drive (i.e. `C:/wad2bin/`

**Comment by KagamineRin271** [comment]
It's still not working, I did everything correctly but it's still giving me the error message

**Comment by KagamineRin271** [comment]
I'm really sorry

**Comment by DarkMatterCore** [comment]
@KagamineRin271 There's no need to apologize. Feel free to hit me up via Discord and I'll help you take a better look.

**Comment by KagamineRin271** [comment]
Is everything not working because I was trying to get Just Dance DLCs?

**Comment by DarkMatterCore** [comment]
> Is everything not working because I was trying to get Just Dance DLCs?

No, that's unrelated to the path length limit error you previously described.

**Comment by KagamineRin271** [comment]
Now it's telling me the same thing but it also says, "the .bin file(s) will be created at the output directory. You can set your SD card root directory as the output directory." And it didn't even make the bin file

**Comment by KagamineRin271** [comment]
![Screenshot_20240209-080620](https://github.com/DarkMatterCore/wad2bin/assets/147155221/19d5d046-afff-46e9-8cb8-cf33ea97a494)

The error code is in the Usage section for some reason

**Comment by KagamineRin271** [comment]
I'm sorry if I can't fix this

**Comment by KagamineRin271** [comment]
> @KagamineRin271 There's no need to apologize. Feel free to hit me up via Discord and I'll help you take a better look.

What's your Discord?

**Comment by KagamineRin271** [comment]
Anyone here? Sorry

**Comment by KagamineRin271** [comment]
> > @KagamineRin271 There's no need to apologize. Feel free to hit me up via Discord and I'll help you take a better look.
> 
> What's your Discord?

Nevermind, I found it

**Comment by DarkMatterCore** [comment]
@KagamineRin271 I'll close this issue. Feel free to reopen it if needed.
